conans/client/tools/scm.py

```diff
@@ -202,7 +202,9 @@ def excluded_files(self):
         return excluded_list
 
     def get_remote_url(self):
-        return self.run("info --show-item url").strip()
+        url = self.run("info --show-item url").strip()
+        revision = self.run("info --show-item revision").strip()
+        return "{url}@{revision}".format(url=url, revision=revision)
```

Comment:
`SVN::get_remote_url` should return just the URL, like the Git implementation, otherwise it would fail [here](https://github.com/jgsogo/conan/blob/cea22a41840056fe0ba06b8bf853d2f7b689a5cc/conans/client/cmd/export.py#L77) when url is a local folder... so I think we have to leave `get_remote_url` as it was and add the peg_revision in this line: https://github.com/jgsogo/conan/blob/cea22a41840056fe0ba06b8bf853d2f7b689a5cc/conans/client/cmd/export.py#L84

Reply:
I just played around with local svn & git repos. 
Git delivers (on windows) something like this:
> T:\rep\my.git

SVN delivers:
> file:///T:/rep/mysvn

Perhaps we should put the "local file" system warning message inside each `get_remote_url` function. So we had just to check for the 'file:///' protocol for svn and could keep the peg revision in the url.

Reply:
Or add another function to check if repository is a local repository.

Reply:
I prefer the second option, we can have inside SCM class a function to check if `remote_url` is, in fact, a local directory. And then, maybe add a `SCM::get_qualified_remote()` that returns `url+peg_rev` in SVN and just `url` in Git? or add a parameter to `get_remote_url` to append or not the peg_rev?

Reply:
I have done option two.